pyaedt/modules/solutions.py

```diff
@@ -938,8 +938,8 @@ def plot_3d(
 
         Returns
         -------
-        :class:`matplotlib.plt`
-            Matplotlib fig object.
+        :class:`matplotlib.figure.Figure`
```

Comment:
```suggestion
        :class:`matplotlib.pyplot.Figure`
```

Reply:
If you look at the source code of matplotlib, matplotlib.pyplot imports Figure from matplotlib.figure.
Do you still want to keep this change though ?

Reply:
@SMoraisAnsys : matplotlib.figure.Figure is correct. You can ignore my comments.